README.md

```diff
@@ -95,8 +98,27 @@ kind get kubeconfig > ./kube.config
 
 ### How does it work
 
-To better depict what `ats` does and how it executes a test scenario, let's have a look at what happens
-when you execute:
+Each run consist of two stages: bootstrapping and test execution.
+
+#### Bootstrapping
+
+`app-test-suite` automates preparation of a cluster used for testing in the following way:
+
+- if you configured your run with `*-tests-cluster-type kind`, a cluster is created with `kind` tool
+- `ats` connects to the target test cluster and runs `apptestctl` - additional tool that deploys components of App Platform
+  (`app-operator`, `chart-operator` and needed CRDs)
+- `ats` deploys `chart-museum`: a simple helm chart registry that will be used to store your chart under test
+- creates `Catalog` CR for the chart repository provided by `chart-museum`
+- creates `App` CR for your chart: as a result, your application defined in the chart is already deployed to the test
+  cluster (you can disable creating this app with `app-tests-skip-app-deploy` option)
```

Comment:
Maybe add a reason why someone would disable that? E. g. when there is a need to create the App CR in a different way, to enforce a certain order.